ads/aja.md

```diff
@@ -21,29 +21,20 @@ limitations under the License.
 ### Basic
 
 ```html
-  <amp-embed width="100" height="100"
+  <amp-ad width="100" height="100"
              type="aja"
              layout="responsive"
-             data-widgetIds="AMP_1,AMP_2">
-  </amp-embed>
+             data-media-id="ZZZZZ">
```

Comment:
Is `data-media-id` also a required param?

Reply:
I made a mistake